src/components/sidebar.tsx

```
@@ -41,97 +43,56 @@ export default function Sidebar({ sidebarOpen, setSidebarOpen, links }: SidebarP
+        transition-transform duration-300 md:translate-x-0 flex flex-col`}
       >
-        {/* Header with logo and close button */}
-        <div className="flex items-center justify-between h-16 border-b border-orange-500 px-4">
+        {/* Header */}
+        <div className="flex items-center justify-between h-16 px-4 border-b border-orange-500">
           <Link href="/dashboard" className="flex items-center gap-2 text-xl font-bold text-white">
             <Store className="w-6 h-6" />
             <span>STORE</span>
           </Link>
-          <button onClick={() => setSidebarOpen(false)} className="md:hidden text-2xl">
+          <button onClick={() => setSidebarOpen(false)} className="md:hidden text-white text-2xl">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)